### PR TITLE
Update CMakeLists.txt to solve the conflictions with onnxruntime

### PR DESCRIPTION
**Pull request by stevenlix**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -278,16 +286,15 @@ target_link_libraries(trt_onnxify PUBLIC nvonnxparser_static ${CUDA_LIBRARIES} $
...
 
-target_link_libraries(onnx2trt PUBLIC nvonnxparser_static ${CUDA_LIBRARIES} ${CMAKE_THREAD_LIBS_INIT} ${CMAKE_DL_LIBS})
+target_link_libraries(onnx2trt PUBLIC ${PROTOBUF_LIB} nvonnxparser_static ${CUDA_LIBRARIES} ${CMAKE_THREAD_LIBS_INIT} ${CMAKE_DL_LIBS})
 
 # --------------------------------
 # API Tests
 # --------------------------------
-
-add_executable(getSupportedAPITest ${API_TESTS_SOURCES})
+add_executable(getSupportedAPITest ${API_TESTS_SOURCES} ${IMPORTER_SOURCES})
```
Comment (kevinch-nv) [inline CMakeLists.txt]: Why is `${IMPORTER_SOURCES} ` added here?

Comment (stevenlix) [reply CMakeLists.txt]: When build getSupportedAPITest, I got errors on protobuf if IMPROTER_SOURCES is not included. Need to investigate why.

Comment (stevenlix) [reply CMakeLists.txt]: The license check got stuck - "license/cla Pending — Contributor License Agreement is not signed yet. ". Who can help with this?

Comment (kevinch-nv) [reply CMakeLists.txt]: Not sure - looks like this is a new thing added for commits into the ONNX repository. @stevenlix It looks like you may have multiple accounts set up on your machine causing the check to ask for more than one user. Can you double check this?

**Pull request closed without merging**